data/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+include(Translations)
+
+if (UBUNTU_BASED_DISTRO)
```

Comment:
You forgot to change this to the new name, hence the branch doesn't work right now.